### PR TITLE
Java: Add changed properties names to the migration guide

### PR DESCRIPTION
**Pull request by maciejwalkowiak**

### CODE REVIEW

**Comment by maciejwalkowiak** [comment]
cc @bruno-garcia

File: src/platforms/java/migration.mdx [hunk 1]
```diff
@@ -109,3 +109,55 @@ _New_:
...
+- `async`
+- `async.shutdowntimeout`
+- `async.gracefulshutdown`
+- `async.queuesize`
+- `async.threads`
+- `async.priority`
+- `compression`
+- `maxmessagelength`
+- `timeout`
+- `factory`
```
Comment (marandaneto) [inline src/platforms/java/migration.mdx:134]: not all of them are available in the new SDK though, I'd separate them, addind a section for deprecated/removed

File: src/platforms/java/migration.mdx [hunk 2]
```diff
@@ -110,13 +110,10 @@ _New_:
...
-### Property Name Changes
+### Removed Properties
 
-Following properties cannot be set anymore through external configuration and have to be set directly on `SentryOptions` object when initializing Sentry:
+Following configuration properties have been removed:
 
-- `mdcTags`
-- `extra`
-- `sample.rate`
 - `buffer.dir`
```
Comment (marandaneto) [inline src/platforms/java/migration.mdx:117]: cacheDirPath

File: src/platforms/java/migration.mdx [hunk 3]
```diff
@@ -110,13 +110,10 @@ _New_:
...
 - `buffer.size`
```
Comment (marandaneto) [inline src/platforms/java/migration.mdx:118]: cacheDirSize

File: src/platforms/java/migration.mdx [hunk 4]
```diff
@@ -130,8 +127,17 @@ Following properties cannot be set anymore through external configuration and ha
 - `async.priority`
 - `compression`
 - `maxmessagelength`
-- `timeout`
 - `factory`
+- `mdcTags`
+- `extra`
```
Comment (marandaneto) [inline src/platforms/java/migration.mdx:132]: is this equivalent to Scope.extra?

**Review by bruno-garcia — APPROVED** [review]

**Comment by bruno-garcia** [comment]
All the notes from @marandaneto showing up as Outdated can be resolved?
Are we ready to merge?

**Comment by maciejwalkowiak** [comment]
If the changes reflect what @marandaneto had in mind then yes

File: src/platforms/java/migration.mdx [hunk 5]
```diff
@@ -173,3 +173,61 @@ Sentry.setExtra("extra", "thing");
...
+#### “In Application” Stack Frames
+
+```java
+stacktrace.app.packages=com.mycompany,com.other.name
+```
+
+_New_:
+
+```java
+in-app-includes=com.mycompany,com.other.name
```
Comment (marandaneto) [inline src/platforms/java/migration.mdx:232]: should we document `in-app-excludes` as well?

**Comment by marandaneto** [comment]
> If the changes reflect what @marandaneto had in mind then yes

this https://github.com/getsentry/sentry-docs/pull/3036#discussion_r572698678 and LGTM

**Review by marandaneto — APPROVED** [review]

**Pull request merged**